Formula/z/zsh-git-prompt.rb

```diff
@@ -1,16 +1,22 @@
 class ZshGitPrompt < Formula
   desc "Informative git prompt for zsh"
-  homepage "https://github.com/olivierverdier/zsh-git-prompt"
-  url "https://github.com/olivierverdier/zsh-git-prompt/archive/refs/tags/v0.5.tar.gz"
-  sha256 "87e5a908369f402e975426ffd61a8800f1c04c0a293f1d4015a6fb1f4408e77d"
+  homepage "https://github.com/zsh-git-prompt/zsh-git-prompt/"
+  url "https://github.com/zsh-git-prompt/zsh-git-prompt/archive/refs/tags/v0.6.tar.gz"
```

Comment:
Is this the official successor or just an active fork?

Reply:
I do not know for sure, so I'm reaching out to @olivierverdier to confirm.

Reply:
@thomasrebele may also have a better sense of the history here.

Reply:
https://github.com/Homebrew/homebrew-core/issues/73961

Reply:
Yes, that comment summarizes the history pretty well. The original creator @olivierverdier had abandoned the project, so I've created a fork. I've created the gh organization to make it possible to transfer the repository to someone else in the future.
I had made the release especially for integrating the new fork into homebrew, see https://github.com/Homebrew/homebrew-core/issues/73961#issuecomment-808721763.

Reply:
Have I abandoned the project? I'd be happy to assist for anything.

Reply:
> Have I abandoned the project? I'd be happy to assist for anything.

At the very least, the [zsh-git-prompt version currently in Homebrew](https://github.com/Homebrew/homebrew-core/blob/master/Formula/z/zsh-git-prompt.rb) appears to not work on macOS Sonoma.

I think there are several questions here:

- Regarding zsh-git-prompt in general: Does @olivierverdier agree with the direction the @thomasrebele and others took in @zsh-git-prompt? Does he wish to hand over the reins, or to incorporate some of the improvements they made?
- Regarding Homebrew, mindful of the answers to the prior questions: Upon which version should the Homebrew formula build? If Homebrew isn't updating the repo location then what do we need to do to fix the current formula?